packages/api/test/e2e/nft.e2e.ts

```diff
@@ -482,6 +481,13 @@ describe('NFTs', () => {
       })
   });
 
+
+  it('Find tokens listing on local with V2', async done => {
+    const allTokens = await api.derive.nft.openCollectionListingsV2('15');
+    expect(allTokens.length).toBe(2);
```

Comment:
can we do more `expect` here for the details of each tokens?

Reply:
yes.. will add more